socketcan_bridge/CMakeLists.txt

```diff
@@ -74,16 +89,20 @@ add_executable(socketcan_bridge_node
 target_link_libraries(socketcan_bridge_node
   topic_to_socketcan
   socketcan_to_topic
+  socketcan_recover_ctrl
   ${catkin_LIBRARIES}
 )
 
+
```

Comment:
whitespace

Reply:
Is there a `.clang-format` file which can be used to avoid this kind of error?